docs-gen/content/ecosystem/software.md

```diff
@@ -0,0 +1,19 @@
+---
+title: "Software"
+date: 2021-07-04T13:31:46+0000
+weight: 3
+---
+
+Various software frameworks exist to work with VSS data. If something is missing here, please [open an issue](https://github.com/GENIVI/vehicle_signal_specification/issues) and tell us.
+
+### W3C VISS2 reference server
+A reference server written in Go for the VISSv2 specification can be found at [https://github.com/UlfBj/W3C_VehicleSignalInterfaceImpl](https://github.com/UlfBj/W3C_VehicleSignalInterfaceImpl).
+
+### KUKSA.val in-vehicle VISS server
+A VISS server written in C++, optimized for containerized deployments on vehicle computers is provided by the [KUKSA project](https://github.com/eclipse/kuksa.val).
+
+### IoT Event Analytics Plattform
+[IoT Event Analytics](https://github.com/GENIVI/iot-event-analytics) is a Complex Event Processing plattform for the Vehicle Edge. The IoTEA tooling can automatically import VSS an data model, and it can optionally use KUKSA.val to ingest live data from a vehicle.
```

Comment:
I wonder about the completeness and maintainability of a list but if you wish to do so then I am aware of two more VISS server implementations. ACCESS have one on github and Aos includes one.

Reply:
Yes, I considered that, which is why I tried to keep it to a sentence or two. The hope is, as the ecosytems grows, people will push to be included. I think the list does not need to be complete (i.e. does not make sense do list anything completely internal here, that can neither be cloned nor bought), as long as we remain open.

I did put a small AOS section, but I left it vague because I lack specifics. Current version:

>  ### AOS
> The [AOS platform](https://aoscloud.io/) can support VSS to access vehicle data.

If anybody can provide a better wording, or additons to the list, happy to include/change